prometheus_blackbox_exporter/tasks/main.yaml

```diff
@@ -0,0 +1,40 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+---
+- name: Install package
+  ansible.builtin.package:
+    name: prometheus-blackbox-exporter
+    state: present
+
+- name: Configure probes
+  ansible.builtin.template:
+    src: blackbox.yml.j2
+    dest: '{{ blackbox_exporter__config_file }}'
+    owner: root
+    group: root
+    mode: '0644'
+  notify: _blackbox_exporter__reload_blackbox_exporter
+
+- name: Configure service
+  ansible.builtin.template:
+    src: default.j2
+    dest: /etc/default/prometheus-blackbox-exporter
+    owner: root
+    group: root
+    mode: '0644'
+  notify: _blackbox_exporter__restart_blackbox_exporter
+
+- name: Start and enable service
+  ansible.builtin.service:
+    name: prometheus-blackbox-exporter
+    state: started
+    enabled: true
+
+- name: Install ferm configuration
+  ansible.builtin.template:
+    src: ferm.conf.j2
+    dest: /etc/ferm/ferm.d/prometheus-blackbox-exporter.conf
```

Comment:
```suggestion
    dest: /etc/ferm/ferm.d/svc-prometheus-blackbox-exporter.conf
```
❓

Reply:
You cannot use `svc-`, because the ferm role owns that namespace and will delete the file

Reply:
Ah, thanks! I missed this because I was running `ansible-playbook --check --tags blackbox-exporter`. I can see the behavior now adding ferm tag. :+1: